src/main/js/components/content/generator/SearchForm.jsx

```diff
@@ -196,7 +220,7 @@ class SearchForm extends React.Component {
                   </tr>
                 </thead>
                 <tbody>
-                  { schedule.map(courseList => (
+                  {schedule.map(courseList => (
```

Comment:
Are the spaces a lint error I'm not getting?

Reply:
I didn't do that manually. It could have possibly been when I pressed 'alt' + 'shift' + 'f' and VS code did an auto formatting thing for me.

Reply:
Follow up: Yes, it was the VS Code auto code formatter.